vital_checks.cpp

```diff
@@ -0,0 +1,42 @@
+// vital_checks.cpp
+#include "./vital_checks.h"
+#include "./warnings.h"
+
+// Constants for vital limits
+const float MAX_TEMPERATURE = 102.0;
+const float MIN_TEMPERATURE = 95.0;
+const float MIN_PULSE_RATE = 60.0;
+const float MAX_PULSE_RATE = 100.0;
+const float MIN_SPO2 = 90.0;
+
+// Function to check if a value is within a given range
+bool isWithinRange(float value, float min, float max) {
+    return (value >= min && value <= max);
+}
+
+// Function to check temperature and display warning if needed
+bool checkTemperature(float temperature) {

```

Comment:
Like the function `isWithinRange`, name bool-return functions with "is". E.g., `IsTemperatureWithinRange`
Then I don't need to refer to the function to clarify its meaning when reading code that calls it.

Reply:
Updated